initializr-service/pom.xml

```diff
@@ -12,6 +12,11 @@
 
 	<properties>
 		<main.basedir>${basedir}/..</main.basedir>
+		<maven.build.timestamp.format>yyyy_MM_dd-HH_mm</maven.build.timestamp.format>
+		<docker.image.prefix>${your-docker-image-prefix}</docker.image.prefix>
+        <docker.image.name>${your-docker-image-name}</docker.image.name>
```

Comment:
Where are these `${your-docker-image-prefix} and ${your-docker-image-name}` and how to use them?

Reply:
from the doc, will guide you to finish this configuration.

Reply:
https://github.com/yaweiw/azure-spring-cloud-playground/blob/master/DEPLOY_TO_AZURE.adoc

Reply:
Did not find the variables above in the doc though. Wonder can we use fixed image prefix and image name here to reduce customization?

Reply:
The doc should be updated due to pom change. The prefix related to ACR repo, cannot be fixed from azure portal.

Reply:
Agree with @sophiaso. ${your-docker-image-prefix} and ${your-docker-image-name} are misleading. What do they refer to? e.g. 
REPOSITORY                     TAG                 IMAGE ID            CREATED             SIZE
demo/cloud-hystrix-dashboard   latest              3ac04a16ab90        3 days ago          132MB

Reply:
ok then. Send PR to change Doc and give an example of what they refer to.

Reply:
This is not only related to the output from `docker images`, but more important for docker distributed images. How can we identify the unique docker images? It is clear if you consider that all docker images compose of `$prefix/$name:$tag`, `$prefix` can be dockerhub or ACR, `$name` is the docker image name.